custom/icds_reports/sqldata.py

```diff
@@ -1111,8 +1120,93 @@ def columns(self):
         columns = self.get_columns_by_loc_level
         agg_columns = [
             DatabaseColumn(
+                'num_children_0_28days_enrolled_for_services',
+                SumColumn('valid_in_month', filters=self.filters + [EQ('age_tranche', 'age_0')]),
+                slug='num_children_0_28days_enrolled_for_services'
+            ),
+            DatabaseColumn(
+                'num_children_28days6mo_enrolled_for_services',
+                SumColumn('valid_in_month', filters=self.filters + [EQ('age_tranche', 'age_6')]),
+                slug='num_children_28days6mo_enrolled_for_services'
+            ),
+            DatabaseColumn(
+                'num_children_6mo1yr_enrolled_for_services',
+                SumColumn('valid_in_month', filters=self.filters + [EQ('age_tranche', 'age_12')]),
+                slug='num_children_6mo1yr_enrolled_for_services'
+            ),
+            DatabaseColumn(
+                'num_children_1yr3yr_enrolled_for_services',
+                SumColumn(
+                    'valid_in_month',
+                    filters=self.filters + [
+                        OR([
+                            EQ('age_tranche', 'age_24'),
+                            EQ('age_tranche', 'age_36')
+                        ])
+                    ]
+                ),
+                slug='num_children_1yr3yr_enrolled_for_services'
+            ),
+            DatabaseColumn(
+                'num_children_3yr6yr_enrolled_for_services',
+                SumColumn(
+                    'valid_in_month',
+                    filters=self.filters + [
+                        OR([
+                            EQ('age_tranche', 'age_48'),
+                            EQ('age_tranche', 'age_60'),
+                            EQ('age_tranche', 'age_72')
+                        ])
+                    ]
+                ),
+                slug='num_children_3yr6yr_enrolled_for_services'
+            ),
+        ]
+        return columns + agg_columns
+
+
+class DemographicsAWCMonthly(ExportableMixin, SqlData):
+    table_name = 'agg_awc_monthly'
+    engine_id = 'icds-test-ucr'
+
+    @property
+    def get_columns_by_loc_level(self):
+        columns = [
+            DatabaseColumn('State', SimpleColumn('state_name'))
+        ]
+        if self.loc_level > 1:
+            columns.append(DatabaseColumn('District', SimpleColumn('district_name'), slug='district_name'))
+        if self.loc_level > 2:
+            columns.append(DatabaseColumn('Block', SimpleColumn('block_name'), slug='block_name'))
+        if self.loc_level > 3:
+            columns.append(DatabaseColumn('Supervisor', SimpleColumn('supervisor_name'), slug='supervisor_name'))
+        if self.loc_level > 4:
+            columns.append(DatabaseColumn('AWC', SimpleColumn('awc_name'), slug='awc_name'))
```

Comment:
@emord Export reports are not paginated we use them only to export data to excel. Probably we must add a async download.